contrib/Docker/conf_files/supervisor/haproxystats-pull.ini

```diff
@@ -0,0 +1,5 @@
+[program:haproxystats-pull]
+command = /usr/bin/haproxystats-pull -f /etc/haproxystats/haproxystats.conf
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
```

Comment:
Can we add a newline at the end?

Reply:
Yes of course, what is your idea?

Reply:
Just pushed a fixed https://github.com/unixsurfer/haproxystats/pull/28/commits/5a47b2bcf954b72cfc9480379ebd369a25441875